src/main/java/com/google/sps/data/PlaceReviews.java

```diff
@@ -99,7 +99,7 @@ public void addRating(double rating) {
    */
   public boolean reviewedBy(String person) {
     // TODO: Use datastore querying to search for this
```

Comment:
I would rather you raise a NonImplementedException and try/catch it wherever you are using this function for now.

It can be confusing if someone sees this function elsewhere and doesn't know it's hardcoded.

Reply:
The dev code wasn't compiling so I just quickly had it return false to get it to work! I didn't know there was a non-implemented exception, I'll keep that in mind for the future / update it in my next PR since this one has been merged!